# This srv file defines the service
# for attaching or detaching an object
# given attach/detach command and armType specified

bool attach
string armType
---
bool success